components/constellation/sandboxing.rs

```diff
@@ -160,7 +160,9 @@ pub fn spawn_multiprocess(content: UnprivilegedContent) -> Result<(), Error> {
     setup_common(&mut child_process, token);
     let _ = child_process
         .spawn()
-        .expect("Failed to start unsandboxed child process!");
+        .expect("Failed to start unsandboxed child process!")
+        .wait()
+        .expect("Failed to wait on unsandboxed child process!");
```

Comment:
I am not really sure about this one, but down bellow we do not wait, so not waiting on child process might be intetnional, cc @wusyong, @jdm who are more familiar with multi-process.

Reply:
Yes, the `.wait` call should be removed. The parent process (main process) and the child (content process) communicate using an ipc-channel established [here](https://github.com/servo/servo/blob/40b1e1767c5487ced71e4baf5ef7507962bbc2f4/components/constellation/sandboxing.rs#L165), so the `.wait` here would block the Constellation thread in the main process forever.

`./mach run -- -M` demonstrates the issue.